app/background.cc

```diff
@@ -183,13 +187,22 @@ int grab_background(std::shared_ptr<background_t> pbkd, int width, int height, c
     if (pbkd->video) {
         // grab frame & frame no. under mutex
         std::unique_lock<std::mutex> hold(pbkd->rawmux);
-        cv::resize(pbkd->raw, out, cv::Size(width, height));
+        cv::Rect crop = bs_calc_cropping(pbkd->raw.cols, pbkd->raw.rows, width, height);
+        cv::resize(pbkd->raw(crop), out, cv::Size(width, height));
         frm = pbkd->frame;
     } else {
-        // resize still image as requested into out
-        cv::resize(pbkd->raw, out, cv::Size(width, height));
-        frm = 1;
-    }
+		if (!pbkd->bg_stored) {
+			// resize still image as requested into out
+			cv::Rect crop = bs_calc_cropping(pbkd->raw.cols, pbkd->raw.rows, width, height);
+			// Under some circumstances we must do the job in two steps!
+			// Otherwise this resize(pbkd->raw(crop), pbkd->raw, ...) may fail.
+			pbkd->raw(crop).copyTo(pbkd->raw);
+			cv::resize(pbkd->raw, pbkd->raw, cv::Size(width, height));
+			pbkd->bg_stored = true;
+		}
+		out = pbkd->raw ;
+		frm = 1;
+	}
```

Comment:
Thank you for fixing this - it was me being lazy to start with... :+1:

Reply:
Some files contain spaces, other tabs. background.cc use spaces. I have first replaced my tabs with spaces as indentation within this file.